packages/amplify-graphql-conversation-transformer/src/__tests__/__snapshots__/amplify-graphql-conversation-transformer.test.ts.snap

```diff
@@ -161,18 +160,18 @@ export function request(ctx) {
     selectionSet,
   };
   const currentMessageId = ctx.stash.defaultValues.id;
+
   const modelConfiguration = {
-  modelId: 'anthropic.claude-3-haiku-20240307-v1:0',
-  systemPrompt: "You are a helpful chatbot. Answer questions to the best of your ability.",
-  inferenceConfiguration: {"temperature":0.5,"topP":0.9,"maxTokens":100},
-};
+    modelId: "anthropic.claude-3-haiku-20240307-v1:0",
+    systemPrompt: "You are a helpful chatbot. Answer questions to the best of your ability.",
+    inferenceConfiguration: {"temperature":0.5,"topP":0.9,"maxTokens":100},
+  };
```

Comment:
Changes are from `JSON.stringify(...)` (`''` --> `""`) and maintaining proper indentation.